nUpdate.WPFUserInterface/UpdateConfiguration.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright © Dominic Beger 2018
```

Comment:
These classes (except UpdaterUI, as it is implemented differently) can be used directly from nUpdate.ProvideTAP. You just need to reference it internally.

Reply:
OK, if i reference the ProvideTAP i can`t initiate an UpdateManager because the compiler says the class is existing in nUpdate.Shared and nUpdateProvideTAP. But because you are using the same Namespaces i can`t say to the compiler what he should use and he will use automatically the nUpdate.Shared.UpdateManager.

Reply:
Okay, that is weird. That is what partial classes should actually be used for. Hm. Can't you tell the compiler that he should use the one from ProvideTAP? But this is actually strange.

Reply:
No, i cant, because you are using the same Namespace in ProvideTAP. You say i should use allways Projectname.Namespace but you have use the Namespace nUpdate.Updating in your ProvideTAP.

Reply:
Ah well, didn't think about that. Then let me think about this. If we can't handle that, we just leave it double-implemented.

Reply:
An Idea is we leave it and Handle that Issue later, maybe in V3?